regression/scripts/airfoil_import/Transition1.txt

```diff
@@ -1,4 +1,4 @@
-Airfoil Transition 1 between/Users/matthewclarke/Documents/SUAVE/SUAVE/regression/scripts/Vehicles/Airfoils/Clark_y.txtand/Users/matthewclarke/Documents/SUAVE/SUAVE/regression/scripts/Vehicles/Airfoils/E63.txt
+Airfoil Transition 1 between/Users/rerha/Desktop/SUAVECODE/SUAVE/regression/scripts/Vehicles/Airfoils/Clark_y.txtand/Users/rerha/Desktop/SUAVECODE/SUAVE/regression/scripts/Vehicles/Airfoils/E63.txt
```

Comment:
can you save this directory as a relative directory? Then it wouldn't change every time.

Reply:
Yes, been meaning to do this. Fixed.